recipes/example/meta.yaml

```diff
@@ -16,6 +16,9 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
So the user/org part won't be correct generally. Should we template it? Note this obvious point?

Reply:
Templating seemed to be :-1: 
https://github.com/conda-forge/staged-recipes/pull/4734#issuecomment-355387903
https://github.com/conda-forge/staged-recipes/pull/4734#discussion_r160053925